examples/payment_channel/lib.rs

```diff
@@ -0,0 +1,475 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+use ink_lang as ink;
+
+#[ink::contract]
+mod payment_channel {
+
+    #[ink(storage)]
+    pub struct PaymentChannel {
+        sender: AccountId,
+        recipient: AccountId,
+        expiration: Timestamp,
+        withdrawn: Balance,
+        close_duration: Timestamp,
+    }
+
+    #[ink(event)]
+    pub struct SenderCloseStarted {
+        expiration: Timestamp,
+        close_duration: Timestamp,
+    }
+
+    impl PaymentChannel {
+        #[ink(constructor)]
+        pub fn new(recipient: AccountId, close_duration: Timestamp) -> Self {
+            Self {
+                sender: Self::env().caller(),
+                recipient,
+                expiration: u64::pow(2, 63) - 1,
+                withdrawn: 0,
+                close_duration,
+            }
+        }
+
+        // sender can escrow funds
```

Comment:
Should be using doc comments here (`///` vs. `//`).

This is an important function, can you expand on the expected usage here?